### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/advanced/advanced-dependencies.md`

### PR DESCRIPTION
**Pull request by kim-sangah**
Add translation to docs/ko/docs/advanced/advanced-dependencies.md

### CODE REVIEW

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+## 매개변수화된 의존성
+
+지금까지 본 모든 의존성은 고정된 함수 또는 클래스입니다.
+
+하지만 여러 개의 함수나 클래스를 선언하지 않고도 의존성에 매개변수를 설정해야 하는 경우가 있을 수 있습니다.
+
+예를 들어, `q` 쿼리 매개변수가 특정 고정된 콘텐츠를 포함하고 있는지 확인하는 의존성을 원한다고 가정해 봅시다.
+
+이때 해당 고정된 콘텐츠를 매개변수화할 수 있길 바랍니다.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/advanced-dependencies.md:11]:
```suggestion
예를 들어, `q` 쿼리 매개변수가 특정 고정된 내용을 포함하고 있는지 확인하는 의존성을 원한다고 가정해 봅시다.

이때 해당 고정된 내용을 매개변수화할 수 있길 바랍니다.
```

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+```Python hl_lines="11"
+{!> ../../docs_src/dependencies/tutorial011_an.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip "팁"
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/advanced-dependencies.md:39]:
```suggestion
/// tip "참고"
```

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+```Python hl_lines="8"
+{!> ../../docs_src/dependencies/tutorial011_an.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip "팁"
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/advanced-dependencies.md:76]:
```suggestion
/// tip "참고"
```

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+```Python hl_lines="17"
+{!> ../../docs_src/dependencies/tutorial011_an.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip "팁"
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/advanced-dependencies.md:112]:
```suggestion
/// tip "참고"
```

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+```Python hl_lines="21"
+{!> ../../docs_src/dependencies/tutorial011_an.py!}
+```
+
+////
+
+//// tab | Python 3.8+ non-Annotated
+
+/// tip "팁"
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/advanced-dependencies.md:157]:
```suggestion
/// tip "참고"
```

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+
+///
+
+```Python hl_lines="20"
+{!> ../../docs_src/dependencies/tutorial011.py!}
+```
+
+////
+
+/// tip "팁"
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/advanced-dependencies.md:169]:
```suggestion
/// tip "참고"
```

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+이 모든 과정이 복잡하게 느껴질 수 있습니다. 그리고 지금은 이 방법이 얼마나 유용한지 명확하지 않을 수도 있습니다.
+
+이 예시는 의도적으로 간단하게 만들었지만, 전체 구조가 어떻게 작동하는지 보여줍니다.
+
+보안 관련 장에서는 이와 같은 방식으로 구현된 유틸리티 함수들이 있습니다.
```
Comment (JaeHyuckSa) [inline docs/ko/docs/advanced/advanced-dependencies.md:175]:
```suggestion
보안 관련 장에서는 이와 같은 방식으로 구현된 편의 함수들이 있습니다.
```

**Review by JaeHyuckSa — CHANGES_REQUESTED** [review]
Hello! @kim-sangah, thank you for your contribute. 👍 There are a few revisions needed. (Related issue: https://github.com/fastapi/fastapi/issues/2017)

**Review by JaeHyuckSa — APPROVED** [review]

File: docs/ko/docs/advanced/advanced-dependencies.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+///
+
+```Python hl_lines="20"
+{!> ../../docs_src/dependencies/tutorial011.py!}
+```
+
+////
+
+/// tip "참고"
```
Comment (alejsdev) [inline docs/ko/docs/advanced/advanced-dependencies.md:169]:
```suggestion
/// tip | 참고
```

**Review by alejsdev — CHANGES_REQUESTED** [review]
Hi @kim-sangah, thanks for your interest in contributing to FastAPI.
However, I've noticed an error in the preview:

image

You can see the translations guidelines for more information here:
https://fastapi.tiangolo.com/contributing/#translations


Thanks!

**Review by alejsdev — APPROVED** [review]
Fixed now! Thanks :nerd_face:

**Comment by alejsdev** [comment]
Thanks for your contribution! :smile: @kim-sangah 
And thanks for your review :nerd_face: @JaeHyuckSa

**Pull request merged**